### PR TITLE
Export more data from dokuwiki metadata

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

**Comment by TheKeymaster** [comment]
@georgms, please review.

File: _test/export.response.test.php [hunk 1]
```diff
@@ -163,4 +165,39 @@ public function test_element_name_is_empty_when_page_has_no_title()
         $name = (string)$names[0];
         $this->assertEmpty($name, 'Expected name in XML should be empty when page has no title.');
     }
+
+    public function test_element_description_is_entire_page_data()
+    {
+        // Generate much text for DokuWiki page
+        $muchContent = '';
+        for ($i=0; $i <= 500; $i++){
```
Comment (georgms) [inline _test/export.response.test.php]: Try `str_repeat()`.

File: _test/export.response.test.php [hunk 2]
```diff
@@ -163,4 +165,39 @@ public function test_element_name_is_empty_when_page_has_no_title()
...
+        $descriptions = $xml->xpath('/findologic/items/item/descriptions/description');
+        $description = (string)$descriptions[0];
+        $this->assertEquals($muchContent, $description, 'Expected description in XML should be equal to the *entire* page content.');
+    }
+
+    public function test_element_summary_is_only_a_part_of_page_data()
+    {
+        $pageId = 'bigpage:muchcontent';
+        $muchContent = '';
+        for ($i=0; $i <= 500; $i++){
```
Comment (georgms) [inline _test/export.response.test.php]: Try `str_repeat()`.

File: DokuwikiXMLExport.php [hunk 3]
```diff
@@ -238,10 +254,9 @@ private function getPageId($pageId)
      */
     private function getAttributesCategory($pageId)
     {
-        $ordernumber = $this->getPageId($pageId);
-
-        $attribute = str_replace('_', ' ', $ordernumber);
-        $attribute = str_replace(':', '_', $attribute);
+        $attribute = str_replace(self::CATEGORY_DELIMITER, ' ', $pageId); // '_' -> ' '
```
Comment (georgms) [inline DokuwikiXMLExport.php]: Always good to provide an explanatory comment for these kind of things. :+1:

**Review by georgms — CHANGES_REQUESTED** [review]
Some minor improvements, but I like this a lot!

File: _test/export.response.test.php (hunk 1 above)
Comment (TheKeymaster) [reply _test/export.response.test.php]: Cool that there is a function for that. Will use it :)

File: DokuwikiXMLExport.php (hunk 3 above)
Comment (TheKeymaster) [reply DokuwikiXMLExport.php]: In the PHPDoc I have added 1 more example and I edited the comments next to the code line.

**Review by georgms — APPROVED** [review]
Well done!

**Pull request merged**